mw-webapp/src/component/confirm/Confirm.stories.tsx

```diff
@@ -34,6 +34,7 @@ const Template: StoryFn<typeof Confirm> = (args) => {
       isOpen={isOpen}
       onOk={() => {}}
       okText="Ok"
+      cancelText="Cancel"
```

Comment:
Multilang

Reply:
It's stories - better  let's dont use multilang